database-portfolio/src/main/webapp/script.js

```diff
@@ -13,42 +13,77 @@
 // limitations under the License.
 //
 
+async function getUserLoginStatus() {
+  const url = '/login';
+  let response = await fetch(url);
+  let text = await response.text();
+  return text;
+}
+
 /** Loads and displays comments according to a limit */
-function loadCommentSection(commentLimit) {
-    // clear previous elements
-    var currentDOM = document.getElementsByClassName('indiv-comments');
-    if (currentDOM != null) {
-        console.log(currentDOM);
-        Array.from(currentDOM).forEach(function (element) {
+async function loadCommentSection(commentLimit) {
+    let userLoginInfo = await getUserLoginStatus();
+    let isLoggedIn = "Y" === userLoginInfo[0];
+
+    // clear previous elements to refresh login status
+    var loginDOM = document.getElementsByClassName('login');
+    if (loginDOM != null) {
+        Array.from(loginDOM).forEach(function (element) {
+            element.remove();
+        })
+    }
+
+    // clear previous elements to refresh comments
+    var commentDOM = document.getElementsByClassName('indiv-comments');
+    if (commentDOM != null) {
+        Array.from(commentDOM).forEach(function (element) {
```

Comment:
Don't use forEach. Recommend standard loop or for...of loop.

Reply:
Also, don't use old fashion anonymous functions "function (element) {}". Instead use, arrow function "element => {}". Why? Because it preserves the "this" keyword should we ever need it.